src/Components/Contact/Contact.module.css

```diff
@@ -0,0 +1,71 @@
+.container {
+  background-color: rgba(203, 47, 101, 0.1);
+  padding: 51px 0 103px 108px;
+  margin-top: 60px;
+}
```

Comment:
Ensure all your containers have the same max-width. Also I think your navigation bar is aligned differently to other sections of your webpage, which makes things seem misaligned. Use the same container style as your navigation.